ansible/inventory/env/group_vars/all.yml

```diff
@@ -551,3 +555,7 @@ sunbird_trampoline_desktop_keycloak_client_id: trampoline-desktop
 # DIAL-service schema
 dial_plugin_container_name: "sunbird-dial-{{env}}"
 dial_service_schema_base_path: "https://{{sunbird_public_storage_account_name}}.blob.core.windows.net/{{dial_plugin_container_name}}/jsonld-schema/local"
+
+### Release-5.0.0 cloud service provider changes for supporting multiple providers ###
+### cloud_service_provider value should be either (azure, aws, gcloud) as per cloud sdk dependency ###
+cloud_service_provider: "azure"
```

Comment:
This will be defined in private repo. Please remove it from here.

Reply:
@keshavprasadms cassandra_isMultiDCEnabled and sunbird_cassandra_consistency_level will also be added to private repo rite?

Reply:
Yes if we want to enable multi dc, we will add it in private repo and override. But by default we will make multi dc as `false` and consistency we will default to `local_quorum`

Reply:
Yes if we want to enable multi dc, we will add it in private repo and override. But by default we will make multi dc as false and consistency we will default to local_quorum